build-support/build-opts.js

```diff
@@ -34,7 +34,7 @@ const replacementStrings = {
 if (codap) {
   replacementStrings.css.push(
     {
-      search: /url\(/g,
+      search: /url\(..\/..\//g,
```

Comment:
I may be missing something here, but
1. Wouldn't those `.`s match any character? It seems like you probably want something like: `/url\(\.\.\/\.\.\//g`, which admittedly looks pretty ridiculous, but 🤷 .
2. Wouldn't this break the other images and fonts that are at `../` paths?
It seems like we need something that catches the problematic font but doesn't change the behavior for everything else. What was the original change that required this change?

Reply:
I agree with (1). Dumb error on my part.
Concerning (2), there aren't others with the "../../" pattern in the styles directory. It was serendipity that the original RE worked at all. It altered other urls, but these are unused in CODAP. Some other process in the build script apparently strips one of the "../" substrings from urls. 

Have uploaded a better fix. It only alters the broken font URLs.